packages/image_picker/image_picker/ios/Classes/image_picker-umbrella.h

```diff
@@ -0,0 +1,10 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Foundation/Foundation.h>
+#import <image_picker/FLTImagePickerImageUtil.h>
+#import <image_picker/FLTImagePickerMetaDataUtil.h>
+#import <image_picker/FLTImagePickerPhotoAssetUtil.h>
+#import <image_picker/FLTImagePickerPlugin.h>
+#import <image_picker/FLTPHPickerSaveImageToPathOperation.h>
```

Comment:
Are this header and the Util headers really intended to be public?

@jmagman Only public headers should be in the umbrella header, right (as with a framework master header)?

Reply:
> @jmagman Only public headers should be in the umbrella header, right (as with a framework master header)?

Correct.  Looks like it was public before this change, which isn't right.

Reply:
How would I solve this the best way? If I simply remove the util headers from the umbrella header I run into issues with the unit tests for those util classes. For example:

```
/packages/image_picker/image_picker/example/ios/RunnerTests/ImageUtilTests.m:17:24: Unknown receiver 'FLTImagePickerImageUtil'; did you mean 'FLTImagePickerPlugin'?
```

Should I create special test headers and add them to the `Test` module in the .modulemap?